subsys/bluetooth/host/conn_internal.h

```diff
@@ -182,12 +182,22 @@ struct bt_conn_tx {
 	void *user_data;
 };
 
+enum acl_data_flags {
+	/** @brief Num Complete Sent from the Host */
+	ACL_DATA_HOST_NUM_COMPLETE_SENT = BIT(0),
+};
+
+BUILD_ASSERT(sizeof(enum acl_data_flags) <= sizeof(uint8_t));
```

Comment:
Can we ensure that an enum is that size without `__packed` or something similar? 

Why do we need to check this?

Reply:
Yeah, the compiler was yelling at me anyways.
We don't need it, I removed it.